.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        elixir-version: [1.10.x, 1.9.x, 1.8.x]
+        include:
+          - elixir-version: 1.10.x
+            otp-version: 22.x
+          - elixir-version: 1.9.x
+            otp-version: 21.x
+          - elixir-version: 1.8.x
+            otp-version: 20.x
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-elixir@v1
+        with:
+          otp-version: ${{matrix.otp-version}}
+          elixir-version: ${{matrix.elixir-version}}
+      - run: mix deps.get
```

Comment:
Should we create a new `make` target for dependencies?

Reply:
Yes, I’ll add [the one we have in elixir-boilerplate](https://github.com/mirego/elixir-boilerplate/blob/master/Makefile#L75-L78) (minus the `npm` part 😄).